subsys/bluetooth/services/dis.c

```diff
@@ -38,6 +47,46 @@ static struct dis_pnp dis_pnp_id = {
 };
 #endif
 
+#if defined(CONFIG_BT_GATT_DIS_SETTINGS)
+#define DIS_STR_MAX
```

Comment:
Where is this used?

Reply:
Oups! Remains from previous version. Will be removed